### ISSUE TITLE
NSClient++ opens 2 the same ports up on install? 

### ISSUE THREAD

**Issue by nagwindmon** [mickem/nscp#101]
Everything seems working as design, but when I did check on ports being opened, there are 2 listed:
 
![2ports](https://cloud.githubusercontent.com/assets/10515775/6048743/8538318a-ac80-11e4-9fea-f0bb4d8cd1fa.png)

Please advice,
Thanks!

**Comment by mathieuchateau** [comment]
hello,

can you check with tcpview
<https://technet.microsoft.com/en-us/sysinternals/bb897437.aspx>or process
explorer <https://technet.microsoft.com/en-us/sysinternals/bb896653.aspx> from
sysinternals (Microsoft) ?
Just to be sure nscp is the owner of this listening port.

you can also use netstat -bn but a bit hard to read



Cordialement,
Mathieu CHATEAU
http://www.lotp.fr

2015-02-04 21:18 GMT+01:00 nagwindmon <notifications@github.com>:

**Comment by mickem** [comment]
Interesting...
I see the same here

![image](https://cloud.githubusercontent.com/assets/1434697/6049133/55b7ff5a-acb5-11e4-8fd8-ec7907c8c314.png)

Strange indeed... wonder if it is a bug in NSClient++ boost asio or windows :)

**Comment by nagwindmon** [comment]
but again, everything looks good in Nagios XI, all metrics are being reported...

**Comment by mickem** [comment]
Yes, nothing I will fix in the intimidate future, but thanks for reporting it...

**Comment by nagwindmon** [comment]
ok, wait a min! it could potentially cause an issue? what is in fact the issue here any way?

**Comment by mickem** [comment]
NSClient++ does open two port but one should be ipv4 and one ipv6 (assuming the machine has ipv6 support).
But as you can see in my screen shot my client has opened 3 ports. So what I think happens is that when NSClient++ open the ipv6 port it accidentally gets an ipv6 and ipv4 port, but since the ipv4 port is already opened the "other port" is defuncs...

But this is just me guessing...

**Comment by nagwindmon** [comment]
here is the screen shot from my pc:
total of 6!

![tcp](https://cloud.githubusercontent.com/assets/10515775/6050639/3b29798c-ac8f-11e4-8ea4-8e472d73057c.png)

**Comment by mickem** [comment]
ohh... Lotus notes... you have my sympathy!

Anyways... will look into this for 0.4.4

**Comment by nagwindmon** [comment]
thanks Mike! Soon we will be migrating to Microsoft outlook 365...
Any way, current version I'm using is NSCP-0.4.1.105-x64.msi and company preparing to deploy to about 4300 Windows machines, should I wait, before it fixed? or since it does not really affecting Agent performance reporting, just go forward and install it?

**Comment by mickem** [comment]
Well, if the issue is present in 0.4.1 which was released a few years ago, and still here today, without anyone ever mentioning it I would say you will probably not notice...

Also presumably you are deploying with a tool, so pushing an update is not that big a deal right?
Worth noticing is upgrading from 0.4.1 to 0.4.3 will be a bigger update then 0.4.3 to 0.4.4 (and presumably later versions)...

Whats the time frame for your deploy?
I can look into it next week and see what I can dig up, then I can give a better estimate of when it will be fixed...

**Comment by nagwindmon** [comment]
Planning to deploy next month, so please do take a look and see what you can find anything, 
and yes we're utilizing SCCM to deploy it, thank you

**Comment by mickem** [comment]
Should have been fixed, if not please reopen
